README.md

```diff
@@ -181,7 +181,6 @@ Note: if you want to train and evaluate on a task, that task must be in both ``p
 We support two modes of adapting pretrained models to target tasks. 
 Setting `transfer_paradigm = finetune` will fine-tune the entire model while training for a target task.
 The mode will create a copy of the model _per target task_.
-If using a pretrained model such as BERT or GPT, be sure to also set the corresponding fine-tune flag, e.g. `bert_fine_tune = 1`.
```

Comment:
If we're deleting these comments, can we add a new comment somewhere that explains how to do the transfer?

Reply:
I think the comments of transfer_paradigm roughly explains the thing?  Is there anything more we need to add there?